README.md

```diff
@@ -89,7 +89,7 @@ Some of the most common built-in matchers are
 - `s.object(fields)`
 - `s.objectWithOnly(fields)`
 - `s.regex(reg)`
-- `s.string()`
+- `s.string(min, max)`
```

Comment:
I'm not sure we should put all the available fields in the signature

I would prefer for us to leave as `s.string(fields)` in the README
But update [Matchers](https://github.com/TabDigital/strummer/blob/master/MATCHERS.md#string) to have an example using the `min` and `max` fields, like what we do for `integer`